industry_benchmarks/utils/data_gathering.py

```diff
@@ -13,6 +14,26 @@
 from kartograf.atom_mapping_scorer import (
     MappingRMSDScorer, MappingShapeOverlapScorer, MappingVolumeRatioScorer,
 )
+import shutil
+
+from industry_benchmarks.utils.tests.test_extract_results import ligand_network
```

Comment:
Industry benchmarks isn't a package, so we can't do imports within it - we need to have these scripts be standalone.